flake.nix

```diff
@@ -0,0 +1,31 @@
+{
+  description = "Cross compile dividat driver";
```

Comment:
nitpick: This does more than cross compile, provides dev environment, etc.

Reply:
[Clarify flake description](https://github.com/dividat/driver/pull/129/commits/c22989ce7bac1aa469bfebfe6859e6400c823d68)